opensrp-giz-malawi/src/test/java/org/smartregister/giz/fragment/EligibleChildrenReportFragmentTest.java

```diff
@@ -0,0 +1,44 @@
+package org.smartregister.giz.fragment;
+
+import android.view.View;
+import android.widget.ProgressBar;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.powermock.reflect.Whitebox;
+import org.smartregister.view.ListContract;
+
+public class EligibleChildrenReportFragmentTest {
+    private EligibleChildrenReportFragment fragment;
+
+    @Before
+    public void setUp() {
+        MockitoAnnotations.initMocks(this);
+        fragment = Mockito.mock(EligibleChildrenReportFragment.class, Mockito.CALLS_REAL_METHODS);
+    }
+
+    @Test
+    public void testLoadPresenterAssignsPresenter() {
+        Assert.assertNull(Whitebox.getInternalState(fragment, "presenter"));
+
+        ListContract.Presenter presenter = fragment.loadPresenter();
+
+        Assert.assertNotNull(Whitebox.getInternalState(fragment, "presenter"));
+        Assert.assertEquals(presenter, Whitebox.getInternalState(fragment, "presenter"));
```

Comment:
You could use a variable for `Whitebox.getInternalState(fragment, "presenter")` so that its more readable.

Reply:
Done